rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -116,6 +117,37 @@ def test_exec(config):
                 raise TestExecError("RGW service restart failed")
             else:
                 log.info('RGW service restarted')
+        if config.gc_verification is True:
+            log.info('making changes to ceph.conf')
+            conf = config.ceph_conf
+            ceph_conf.set_to_ceph_conf('global', ConfigOpts.bluestore_block_size,
+                                       str(conf.get('bluestore_block_size', 1549267441664)))
+            ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_gc_max_queue_size,
+                                       str(conf.get('rgw_gc_max_queue_size', 367788)))
+            ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_gc_processor_max_time,
+                                       str(conf.get('rgw_gc_processor_max_time', 3600)))
+            ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_gc_max_concurrent_io,
+                                       str(conf.get('rgw_gc_max_concurrent_io', 10)))
+            ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_objexp_gc_interval,
+                                       str(conf.get('rgw_objexp_gc_interval', 10)))
+            ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_gc_max_trim_chunk,
+                                       str(conf.get('rgw_gc_max_trim_chunk', 32)))
+            ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_gc_obj_min_wait,
+                                       str(conf.get('rgw_gc_obj_min_wait', 10)))
+            ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_gc_processor_period,
+                                       str(conf.get('rgw_gc_processor_period', 10)))
+            log.info('trying to restart services ')
+            srv_restarted = rgw_service.restart()
+            time.sleep(30)
+            if srv_restarted is False:
+                raise TestExecError("RGW service restart failed")
+            else:
+                log.info('RGW service restarted')
+            # Delete gc queue
+            pool_name = utils.exec_shell_cmd('ceph df |awk \'{ print $1 }\'| grep rgw.log')
+            pool_name = pool_name.replace("\n", "")
+            utils.exec_shell_cmd('for q in {0..31}; do rados rm gc.${q} -p %s -N gc; done' % pool_name)
```

Comment:
your executing a for loop here. how about adding it in python and not use shell cmd here. there is room for improvement. this is not a ceph-command i think.